docs/GettingStarted.md

```diff
@@ -1401,6 +1401,7 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 | `analytics_enabled` | Enable analytics for spans produced by this integration. `true` for on, `nil` to defer to global setting, `false` for off. | `false` |
 | `client_service_name` | Service name used for client-side `sidekiq` instrumentation | `'sidekiq-client'` |
 | `service_name` | Service name used for server-side `sidekiq` instrumentation | `'sidekiq'` |
+| `log_job_arguments` | Enable logging of job arguments. `true` for on, `false` for off. | `false` |
```

Comment:
To avoid confusion with logging features, I think it would be better to substitute `log` for `tag`. You can shorten `arguments` to `args`. Also is it redundant to say "job arguments" when Sidekiq always deals with jobs?

I would suggest a rename to either `job_args` or `args` depending on your answer to that question, then update all the other parts of code to match.

Reply:
That makes sense, changing it to `tag_args`.